services/venice-controller/src/main/java/com/linkedin/venice/controller/stats/AdminConsumptionStats.java

```diff
@@ -81,21 +79,12 @@ public AdminConsumptionStats(MetricsRepository metricsRepository, String name) {
     adminConsumeFailCountSensor = registerSensor("failed_admin_messages", new Count());
     adminConsumeFailRetriableMessageCountSensor = registerSensor("failed_retriable_admin_messages", new Count());
     adminTopicDIVErrorReportCountSensor = registerSensor("admin_message_div_error_report_count", new Count());
-    registerSensor("failed_admin_message_offset", new Gauge(() -> adminConsumptionFailedOffset));
+    registerSensor(new AsyncGauge((c, t) -> adminConsumptionFailedOffset, "failed_admin_message_offset"));
     adminConsumptionCycleDurationMsSensor =
         registerSensor("admin_consumption_cycle_duration_ms", new Avg(), new Min(), new Max());
-    pendingAdminMessagesCountSensor = registerSensor(
-        "pending_admin_messages_count",
-        new Gauge(() -> pendingAdminMessagesCountGauge),
-        new Avg(),
-        new Min(),
-        new Max());
```

Comment:
Similar question here: let's confirm if avg, min, max for admin message count monitoring is not needed anymore.

Reply:
No dashboard or alerts are using them. I believe these 3 metrics types make no sense here. We just need to know how many pending admin messages are at this moment.